docs/proposals/scheduledjob.md

```diff
@@ -0,0 +1,113 @@
+# ScheduledJob Controller
```

Comment:
If you guys don't mind, I prefer sticking with Scheduled. The main reason for it is I don't want this to be mistaken as a linux cron. Especially that I'm hoping to define the actual schedule using ISO8601.